terraform/daily_snapshot/main.tf

```diff
@@ -29,7 +29,7 @@ module "daily_snapshot" {
 
   # Configure service:
   name              = "forest-snapshot"       # droplet name
-  size              = "so-2vcpu-16gb"         # droplet size
+  size              = "s-4vcpu-16gb-amd"      # droplet size
```

Comment:
Shall we make the exact change in the sync check? Or anywhere else?

Reply:
`s-4vcpu-16gb-amd` is faster but only has 200GiB of storage. It should be enough for the sync check unless you're doing strange things with the snapshots. :)

Reply:
In the past 14 days, we never exceeded 40% of the 300 GB disk, so we should be good for the sync check.
![image](https://github.com/ChainSafe/forest-iac/assets/9642092/0377129f-192c-40d7-bc09-7a9680fbce73)